### ISSUE TITLE
Add heading examples

### ISSUE THREAD

**Issue by IonicaBizau** [Toddsqui/learning-markdown#1]
Like we did for [code blocks](https://github.com/Toddsqui/learning-markdown#code-in-markdown), add examples for Markdown headings. Use Markdown ```` ```md ```` code blocks to format the code correctly. 

Add:

 - heading 1
 - heading 2
 - heading 3
 - heading 4
 - heading 5
 - heading 6

**Comment by Toddsqui** [comment]
```
# Header1
## Header 2
### Header 3
#### Header 4
##### Header 5
###### Header 6
```

**Comment by Toddsqui** [comment]
I think this is what you wanted.

**Comment by IonicaBizau** [comment]
@Toddsqui Exactly, but make sure you add this in README.md :memo:

**Comment by IonicaBizau** [comment]
@Toddsqui Edit the file locally (using Atom) and then commit the changes. As commit message you can use something like `Add headings example. Fixed #1`–using `fixed #1` will close this issue. :smile:

**Comment by Toddsqui** [comment]
@IonicaBizau I think I did it! Check the file!
